meerkat_frontend/src/files/pages/consultations.html

```diff
@@ -0,0 +1,133 @@
+<div class="top-bar row">
+    <div class="location-box box">
+        <a href="#menu-toggle" id="menu-toggle">
+            <div class="location-box__label">
+                <div id="location_box"></div>
+                <div id="location-title" class="location-title"></div>
+            </div>
+            <span class="glyphicon glyphicon-chevron-right"></span>
+            <span class="glyphicon glyphicon-chevron-left hidden"></span>
+        </a>
+    </div>
+    <div class="less-padding-col tab-title hidden-xs col-sm-5 col-md-5" id="title">
+    </div>
+    <div id="epi-week-title" class="col-xs-12 col-sm-12 col-md-3 pull-right box ">
+    </div>
+</div>
+
+<div class="demographics toggled" id="wrapper">
```

Comment:
The class should probably be "consultations" instead of "demographics", but it doesn't matter that much.